commit 5dcb5ef125907d09806509a9db8c6705041e0026
Author: Mark Pearson <mpearson-lenovo@squebb.ca>
Date:   Thu Oct 24 15:55:24 2024 -0400

    platform/x86: think-lmi: Multi-certificate support
    
    Lenovo are adding support for both Admin and System certificates to
    the certificate based authentication feature
    
    This commit adds the support for this.
    
    Signed-off-by: Mark Pearson <mpearson-lenovo@squebb.ca>
    Link: https://lore.kernel.org/r/20241024195536.6992-4-mpearson-lenovo@squebb.ca
    [ij: Added #include <linux/array_size.h> + comment grammar fix]
    Reviewed-by: Ilpo Järvinen <ilpo.jarvinen@linux.intel.com>
    Signed-off-by: Ilpo Järvinen <ilpo.jarvinen@linux.intel.com>

diff --git a/drivers/platform/x86/think-lmi.h b/drivers/platform/x86/think-lmi.h
index 4728f40143a3..f267d8b46957 100644
--- a/drivers/platform/x86/think-lmi.h
+++ b/drivers/platform/x86/think-lmi.h
@@ -41,6 +41,10 @@ enum save_mode {
 };
 
 /* password configuration details */
+#define TLMI_PWDCFG_MODE_LEGACY    0
+#define TLMI_PWDCFG_MODE_PASSWORD  1
+#define TLMI_PWDCFG_MODE_MULTICERT 3
+
 struct tlmi_pwdcfg_core {
 	uint32_t password_mode;
 	uint32_t password_state;